src/pages/App.tsx

```diff
@@ -122,7 +125,6 @@ export default function App() {
               <Route exact strict path="/migrate/v2" component={MigrateV2} />
               <Route exact strict path="/migrate/v2/:address" component={MigrateV2Pair} />
 
-              <Route exact strict path="/create-proposal" component={CreateProposal} />
```

Comment:
I like that you've moved this under `/vote`, but it will no longer match. You'll need to add a redirect in App.tsx from /create-proposal -> /vote/create-proposal (or /vote/propose, or /vote/create), and change the route under `/vote` to the redirected-to path.